tokio/src/runtime/task/mod.rs

```diff
@@ -155,7 +155,14 @@ cfg_rt_multi_thread! {
     pub(super) use self::inject::Inject;
 }
 
+#[cfg(all(feature = "rt", any(tokio_unstable, test)))]
+mod abort;
 mod join;
+
+#[cfg(all(feature = "rt", any(tokio_unstable, test)))]
```

Comment:
I would prefer to *not* expose unstable features w/o the `tokio_unstable` flag. Instead, could unstable tests be conditionally enabled w/ `#[cfg(tokio_unstable)]`?

Reply:
Alternatively, if it is a question of visibility for unit tests, maybe this can just be `pub(crate)` when test?

Reply:
that's what i did initially, @Darksonn suggested this instead in https://github.com/tokio-rs/tokio/pull/4530#discussion_r813728231. i can change it back if you prefer?

Reply:
The `runtime::task` mod isn't public itself, so yeah it isn't a big deal.